lib/tree.rb

```diff
@@ -1,38 +1,59 @@
 class NoApplesError < StandardError; end
 
 class AppleTree
-  attr_#fill_in :height, :age, :apples, :alive
+  attr_accessor :height, :age, :apples, :alive
 
   def initialize
+    @height = 1
+    @age = 0
+    @apples = []
+    @alive = true
   end
 
   def age!
+    @age += 1
+    if @age >= 100 then @alive = false end
+    if @alive == true
```

Comment:
This can be shortened to just @alive because if it's true, it will resolve the if block